src/FsAutoComplete.Core/Commands.fs

```diff
@@ -24,7 +25,7 @@ type LocationResponse<'a,'b> =
 [<RequireQualifiedAccess>]
 type HelpText =
     | Simple of symbol: string * text: string
-    | Full of symbol: string * tip: FSharpToolTipText * moreInfo: option<string * int * int * string>
+    | Full of symbol: string * tip: FSharpToolTipText * textEdits: CompletionNamespaceInsert option
```

Comment:
I turned this into a full type so that I'd have a better idea of what the fields meant later on.